browser.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
-const isReachable = require('is-reachable');
-const hostnames = require('./hostnames');
+
+const isIp = require('is-ip');
+const publicIp = require('public-ip');
 
 module.exports = options => {
-	options = options || {};
-	return isReachable(options.hostnames || hostnames);
+	options = Object.assign({timeout: 5000}, options);
+
+	return publicIp.v4(options).then(ip => {
+		return isIp.v4(ip);
```

Comment:
Hmm yeah this can potentially be move to `public-ip`, and we only check for a truthy value here.